components/hab/src/exec.rs

```diff
@@ -35,9 +50,12 @@ const INTERNAL_TOOLING_CHANNEL_ENVVAR: &str = "HAB_INTERNAL_BLDR_CHANNEL";
 /// # Notes on Package Installation
 ///
 /// By default, Habitat will install packages from the `stable`
-/// channel. However, if you'd rather use unstable (particularly if
-/// you're developing Habitat), you'll need to set the
-/// `INTERNAL_TOOLING_CHANNEL_ENVVAR` appropriately.
+/// channel or will use a channel specific to a particular context (eg
+/// `hab sup run --internal-channel`) passed with the `channel`
```

Comment:
I don't think we have `--internal-channel` yet, right?

Reply:
Oops, thanks! [7c093d3](https://github.com/habitat-sh/habitat/pull/7871/commits/7c093d36863454f4f5da4afca50687efefe87d65)